ui/src/jobs/list/ListEnsemblingJobsView.js

```diff
@@ -78,19 +76,21 @@ export const ListEnsemblingJobsView = (props) => {
   const onRowClick = (item) => props.navigate(`./${item.id}/details`);
 
   return (
-    <EuiPage>
-      <EuiPageBody>
-        <EuiPageHeader>
-          <EuiPageHeaderSection>
-            <PageTitle title="Ensembling Jobs" />
-          </EuiPageHeaderSection>
-          <EuiPageHeaderSection>
-            <EuiButton fill disabled href={"jobs/create"}>
-              Submit Job
-            </EuiButton>
-          </EuiPageHeaderSection>
-        </EuiPageHeader>
-        <EuiPageContent>
+    <EuiPageTemplate restrictWidth={restrictWidth} paddingSize={paddingSize}>
+      <EuiSpacer size="l" />
```

Comment:
Just curious, since I see it at quite a few instances where `EuiPageTemplate` is being used. What do these `EuiSpacer` at the start and end of a `EuiPageTemplate` achieve?

Reply:
It's to simulate the padding that we've removed from `EuiPageTemplate` (we're passing `"none"` as the `paddingSize` prop). The large `EuiSpacer`s at the top and bottom of a page adds some semblance of a padding at these places respectively, while the medium one just above an `EuiPageTemplate.Section` adds some space before the `EuiPageTemplate.Header` above it. See [here](https://github.com/caraml-dev/turing/pull/229#discussion_r959318467) for the effect I was trying to create